src/openms/include/OpenMS/KERNEL/RangeManager.h

```diff
@@ -69,28 +69,39 @@ namespace OpenMS
         throw Exception::InvalidRange(__FILE__, __LINE__, OPENMS_PRETTY_FUNCTION, "Invalid initialization of range");
     }
 
+    /// Copy C'tor
+    RangeBase(const RangeBase& rhs) = default;
+
+    /// Assignment operator
+    RangeBase& operator=(const RangeBase& rhs)
+    {
+      min_ = rhs.min_;
+      max_ = rhs.max_;
+      return *this;
+    }
+
     /// make the range empty, i.e. isEmpty() will be true
     void clear()
     {
       *this = RangeBase(); // overwrite with fresh instance
     }
 
-    /// is the range empty (i.e. default constructed or cleared using clear())?
+    /// is the range empty (i.e. min > max)?
     bool isEmpty() const
-    { // invariant: only possible when default constructed or clear()'ed
+    {
       return min_ > max_; 
     }
 
     /// is @p value within [min, max]?
     bool contains(const double value) const
     {
-      return min_ <= value && value <= max_;
+      return (min_ <= value) & (value <= max_);
```

Comment:
why? I think some guidelines discourage  the use of & vs && for boolean expressions